src/app/shared/palette/date/write-date-field.html

```diff
@@ -8,6 +8,7 @@
     </legend>
 
     <cut-date-input [id]="id()"
+                    [dateTime]="isDateTime()"
```

Comment:
wonder whether this property is really needed. Can't the cut-date-input infer it's a date time by looking at the presence of the time part in the raw string?